src/commands/synthetics/index.ts

```diff
@@ -1,5 +1,5 @@
 export {CiError, CriticalError} from './errors'
 export * from './interfaces'
 export {DefaultReporter} from './reporters/default'
-export {executeTests} from './run-test'
+export {executeTests, getApiHelper} from './run-test'
```

Comment:
Why is it necessary to export `getApiHelper`? from a library point of view, it is rather internal

Reply:
Stopped exporting `getApiHelper()` in favor of utils `fetchTest()` function